src/themes/dojo/comboBox.m.css

```diff
@@ -1,55 +1,103 @@
 @import './variables.css';
 
 .root {
-	position: relative;
-	display: inline-block;
 	box-sizing: border-box;
-	width: var(--form-field-width);
+	color: var(--color-text-primary);
+	display: inline-block;
+	font-size: var(--font-size-base);
+	line-height: var(--line-height-base);
+	position: relative;
+	width: 100%;
 }
 
 .root * {
 	box-sizing: border-box;
 }
 
-.controls {
-	position: relative;
+.root input {
+	line-height: var(--line-height-base);
+	min-width: 200px;
+	padding: var(--spacing-regular) calc(var(--spacing-regular) * 6) var(--spacing-regular) var(--spacing-regular);
+	width: 100%;
+}
+
+.trigger {
+	appearance: none;
+	background-color: transparent;
+	border-bottom: 0;
+	border-left: var(--border-width) solid var(--color-border);
+	border-radius: 0;
+	border-right: 0;
+	border-top: 0;
+	bottom: var(--border-width);
+	color: var(--color-text-faded);
+	cursor: pointer;
+	display: block;
+	font-size: inherit;
+	font-size: var(--font-size-base);
+	line-height: var(--line-height-base);
+	outline: 0;
+	padding: var(--spacing-regular) calc(var(--spacing-regular) + 4);
+	position: absolute;
+	right: 0;
+	top: var(--border-width);
+	z-index: 1;
 }
 
-.input {
+.dropdown {
+	background-color: var(--color-background);
+	border: var(--border-width) solid var(--color-border);
+	box-shadow: var(--box-shadow-dimensions-small) var(--color-box-shadow);
+	opacity: 0;
+	position: absolute;
+	transform-origin: 0 0;
+	transform: rotateX(90deg);
+	transition: all var(--transition-duration) var(--transition-easing);
 	width: 100%;
-	padding-right: 2em;
+	z-index: 2;
```

Comment:
Agreed, good catch. I will do this as part of a separate PR, this will effect SlidePane and Dialog (at least) once they land.